project/SparkRedshiftBuild.scala

```diff
@@ -52,6 +52,7 @@ object SparkRedshiftBuild extends Build {
         "Spark 1.5.0 RC2 Staging" at "https://repository.apache.org/content/repositories/orgapachespark-1141",
       libraryDependencies ++= Seq(
         "com.amazonaws" % "aws-java-sdk-core" % "1.9.40" % "provided",
+        "com.amazonaws" % "aws-java-sdk-s3" % "1.9.40" % "provided",
```

Comment:
I originally marked the AWS SDK as provided for two reasons, though I wasn't 100% sure it's the right thing to do at the time, so we may want to revise. Here's what was bothering me about making them required though:

* On Amazon EMR, which now officially supports Spark pre-installed, the AWS SDK is already on the classpath. EMRFS is also on the classpath, so everything should "just work" with no extra dependencies.
* If you're using S3 and you're not on EMR, you're likely going to be needing `hadoop-aws` instead for the `S3NativeFileSystem` implementation, which has a dependency on the AWS SDK.
* These dependencies are not especially lightweight, and I often find they're a reliable source of dependency hell. I seem to recall them both pulling in Joda, Jackson and Guava, with version ranges/wildcards and all kinds of fun stuff that will no-doubt fight with your Hadoop setup.